apis/orgpolicy/v1alpha1/policy_types.go

```diff
@@ -176,3 +176,12 @@ type PolicySpec struct {
 	// +kcc:proto:field=google.cloud.orgpolicy.v2.PolicySpec.reset
 	Reset *bool `json:"reset,omitempty"`
 }
+
+// +kcc:observedstate:proto=google.cloud.orgpolicy.v2.PolicySpec
```

Comment:
Why?  This looks identical to the generated version

Reply:
the type.generated.go cannot generate this function. 

There's some flaws in the type-generator when we distinguish the Spec and ObservedState from the same proto.

Since we moved the spec part "PolicySpec" from types.generate.go to this manual file, and it has the marker `+kcc:observedstate:proto=google.cloud.orgpolicy.v2.PolicySpec`. The type generator skips the  ObservedState part as well.

I can fix it in a follow up PR, but because we want to enforce the generate.sh. This function has to be in the manual types.go

Reply:
OK, no worries, I assumed that because it was previously in generated that it was generated.  If that's not the case, then this makes sense!